enqueue/enqueue-bundle/0.3/etc/packages/test/enqueue.yaml

```diff
@@ -0,0 +1,6 @@
+enqueue:
+    transport:
+        default: 'null'
+        'null': ~
```

Comment:
Maybe, but not now. That's a huge change and would take time. I'll do it later on. 

Is it a blocker?